waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -236,9 +231,6 @@ genIndexJs spec =
       )
   where
     maybeSetupJsFunction = AS.App.Client.setupFn =<< AS.App.client (snd $ getApp spec)
-    maybeSetupJsFnImportDetails = getJsImportDetailsForExtFnImport extClientCodeDirInWebAppSrcDirP <$> maybeSetupJsFunction
+    maybeSetupJsFnImportDetails = getClientJsImport [reldirP|./|] <$> maybeSetupJsFunction
```

Comment:
How is it that you dropped the `extClientCodeDirInWebAppSrcDirP`?

Reply:
We construct the path to the `web-app/ext-src` inside of the `getClientJsImport` so we don't need it anymore. 

https://github.com/wasp-lang/wasp/pull/956/files#diff-883fead5eb518073046ab034da6eb5e9918a6544075416dd8c51ef415db098e5R15